sockeye/inference.py

```diff
@@ -986,6 +993,15 @@ def _beam_search(self,
                                dtype='int32')
         sequences[:, 0] = self.start_id
 
+        # Beam history
+        beam_histories = []
+        for _ in range(self.batch_size):
+            beam_histories.append({"scores": [],
+                                 "normalized_scores": [],
```

Comment:
why do we need normalized and unnormalized scores?

Reply:
I was going to add both to the beam visualisation, it's interesting to see how normalization rescores the hypothesis. I imagine it could help to see better why a certain path was chosen due to normalization. What do you think?

Reply:
the reason I am asking is that you then need to unnormalize twice, once in-place in the existing code, and once for the beam_history logging.
I would say its fine to have as long as we completely disable the logging by default and only turn it on via a CLI flag / output-handler type.